templates/membership/join.html.twig

```diff
@@ -21,11 +21,31 @@
 
 {% block javascripts %}
     {% import 'javascript.js.twig' as js %}
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
     <script type="text/javascript">
         Kernel.onLoad(function() {
             App.runJoin('{{ form.vars.form_type_class|e('js') }}');
             {{ js.address(form.address) }}
         });
+
+        $( document ).ready(function() {
```

Comment:
maybe we can move this code in a partial template too and include it here (for the same goal, avoid duplication)?